frontend/packages/kubevirt-plugin/package.json

```diff
@@ -37,7 +37,8 @@
       "kubevirt-gating": [
```

Comment:
@omril-redhat hi, when do we run the "kubevirt-gating" tests ?

Reply:
I'm not sure, but suppose to be tests that are not using expect lib.

Reply:
@irosenzw hi, do you know when we run this tests ?

Reply:
@yaacov, yes. whenever you change something inside the `kubevirt-plugin` dir in openshift/console.
you can see the config here:  https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml#L284